src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Extensions/JTokenExtensions.cs

```diff
@@ -73,7 +73,7 @@ internal static PSObject ToPsObject(this JToken jtoken, string objectType = null
             foreach (var property in jobject.Properties())
             {
                 psObject.Properties.Add(new PSNoteProperty(
-                    name: JTokenExtensions.ConvertToPascalCase(propertyName: property.Name), 
+                    name: property.Name,
```

Comment:
Can we please add a test for this? Feel free to edit one of our existing resources tests to add a check for this.